homeassistant/components/zwave_js/light.py

```diff
@@ -369,13 +389,21 @@ def _calculate_color_values(self) -> None:
                     self._max_mireds
                     - ((cold_white / 255) * (self._max_mireds - self._min_mireds))
                 )
+                # White channels turned on, set color mode to color_temp
+                self._color_mode = COLOR_MODE_COLOR_TEMP
```

Comment:
zwave allows all 5 channels in a 5 channel light to be on simultaneously. Unless blocked by zwaev_js, we might want to check which channels are on:
if r, g, b are all zero and at least one white channel is on -> COLOR_MODE_COLOR_TEMP
if at least one of r, g, b is non zero and at least one white channel is on -> COLOR_MODE_RGBWW